resharper/src/resharper-unity/CSharp/Daemon/Errors/CSharpErrors.xml

```diff
@@ -46,6 +46,13 @@
         <Description>Parameter should be derived from `Component`.</Description>
       </Tag>
 	  
+
+      <Tag externalName="InefficientPropertyAccessWarning.HIGHLIGHTING_ID"
+           default="WARNING">
+        <Title>Access properties of built-in component is not efficient. Cache value</Title>
+        <Description>Access properties of built-in component is not efficient due to calling native methods.</Description>
```

Comment:
Built-in component property access results in native method calls and should be cached